src/app/lib/layoutGrid.js

```diff
@@ -234,6 +234,11 @@ export const nestedGridItemLargeCss = css`
       ${props =>
         specifiedOrMaximum(props.gridSpan.group3 || props.gridSpan.default, 6)};
   }
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
```

Comment:
For the Large Nested Grid, I don't think we'll need to add Group 2, since it should be the same as the values for Group 3.